uber/models.py

```diff
@@ -1195,7 +1195,7 @@ def badge_cost(self):
             return c.get_attendee_price(registered)
 
     @cost_property
-    def discount(self):
+    def age_discount(self):
```

Comment:
This is an unrelated change - `discount` is way too generic a name for a property in the core system!